spec/workers/event_worker_spec.rb

```diff
@@ -14,7 +17,47 @@
   #   end.to change(described_class.jobs, :size).by(1)
   # end
 
-  pending 'can process an event'
-  pending 'can retry 3 times'
-  pending 'can clean up when errors are raised'
+  before do
+    stub_request(:post, 'https://myfakeoutbound.com/')
+      .with(headers: { 'Accept' => '*/*', 'User-Agent' => 'Ruby' })
+      .to_return(status: 200, body: { data: 'stubbed response' }.to_json, headers: {})
+  end
+
+  context 'Event worker' do
+    let(:event) { create :event }
+
+    it 'can process an event' do
+      expect(event.reload.completed).to eq false
+
+      EventWorker.new.perform event.id
```

Comment:
I missed this in my previous review. Since this is testing the whole system together, you should pull the event obj from the db and test that all data inputs are how they should be. This is just testing "in a perfect world, we don't raise errors".

Reply:
I updated this spec. I'm now testing that the response object is as expected. I didn't test the request object because I figured we test that functionality elsewhere & it's just values we're setting up ourselves. Let me know what you think!